src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -0,0 +1,119 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateSet('install', 'ensure-permission')]
+    [string]
+    $command,
+
+    [string]
+    $downloadFrom,
+
+    ## Not doing anything with this parameter yet
+    [string]
+    $sessionId,
+
+    [string]
+    $serviceName = "Microsoft IIS Administration",
+
+    [string]
+    $iisAdminOwners = "IISAdminAPIOwners"

```

Comment:
We shouldn't use abbreviation in our group names.

This should be "IIS Administration API Owners"